tools/test.c

```diff
@@ -532,26 +563,162 @@ try_boot(void)
 	struct scpi_msg msg;
 	long boot_time;
 
+	/* Seeing "SCP ready" means the firmware booted successfully. */
 	log(LOG_INFO, "Waiting for firmware...");
 	test_begin(TEST_BOOT);
 	scpi_prepare_msg(&msg, SCPI_CMD_SCP_READY);
 	test_serve_reply(&msg, &boot_time);
 	log(LOG_INFO, "Firmware booted in %ld ns", boot_time);
 	test_complete(TEST_BOOT);
 
+	/* Send an invalid command to test basic message processing. */
 	test_begin(TEST_INTERRUPT);
```

Comment:
This test as well as TEST_SCP_CAP should be in a separate function.

Reply:
Done